aomp-flang/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# Maintainer: Torsten Keßler <t dot kessler at posteo dot de>
+
+pkgname=aomp-flang
+pkgdesc='Fortran component for AOMP'
+_pkgver=11.6-1
+pkgver=11.6.1
+pkgrel=1
+arch=('x86_64')
+url='https://github.com/ROCm-Developer-Tools/flang'
+license=('custom:Apache 2.0 with LLVM Exception')
+depends=(aomp-llvm aomp-openmp z3)
+makedepends=(cmake python gcc-fortran)
+source=("${pkgname}-${pkgver}.tar.gz::$url/archive/aomp-$_pkgver.tar.gz")
+sha256sums=('f94e6f72428816fdcda0a99de1e8d3405388a7f30bab16f6a727cab56b2de489')
+_dirname="$(basename "$url")-$(basename ${source[0]} .tar.gz)"
+
+build() {
+    CC=/opt/rocm/aomp/bin/clang \
+    CXX=/opt/rocm/aomp/bin/clang++ \
+    cmake -Wno-dev -B build \
+          -S "$_dirname" \
+          -DCMAKE_INSTALL_PREFIX='/opt/rocm/aomp' \
+          -DCMAKE_BUILD_TYPE=Release \
+          -DLLVM_ENABLE_ASSERTIONS=ON \
+          -DLLVM_TARGETS_TO_BUILD='AMDGPU;X86' \
+          -DLLVM_CONFIG='/opt/rocm/aomp/bin/llvm-config' \
+          -DFLANG_OPENMP_GPU_AMD=ON \
+          -DFLANG_OPENMP_GPU_NVIDIA=ON \
+          -DLLVM_INSTALL_TOOLCHAIN_ONLY=ON \
```

Comment:
Are we sure we only need the toolchain here?
Could this be the cause for fortran linker errors.

Reply:
Nevermind I see upstream also sets this here
https://github.com/ROCm-Developer-Tools/aomp/blob/dd9fb8d82a94c98302fc104a0f9a8e37777d9797/bin/build_flang.sh